### PR TITLE
add version matrix

### PR DESCRIPTION
**Pull request by crodriguezvega**
## Description

Version matrix golang/ibc-go/SDK/tendermint

closes: #1041

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

File: RELEASES.md [hunk 1]
```diff
@@ -97,6 +97,21 @@ As rule of thumb, the following changes will **NOT** be automatically accepted i
 * **API-breaking changes**, i.e. changes that prevent client applications to *build without modifications* to the client application's source code.
 * **CLI-breaking changes**, i.e. changes that require usage changes for CLI users.
 
+## Version matrix
```
Comment (crodriguezvega) [inline RELEASES.md:100]: I added it to RELEASES.md, but open to suggestions to have it another place.

We can add later on a link to the matrix on the README.md when working on #1747.

**Review by chatton — APPROVED** [review]
LGTM,  I think RELEASES.md is a good place for it 👍 , we just need to make sure that we keep it up to date as part of the release process.

File: RELEASES.md (hunk 1 above)
Comment (charleenfei) [reply RELEASES.md:100]: sounds good for now, we can always iterate if it needs moving later :)

**Review by charleenfei — APPROVED** [review]
nice!

**Review by damiannolan — APPROVED** [review]

**Review by colin-axner — APPROVED** [review]

**Pull request merged**